test/message-service.test.js

```diff
@@ -0,0 +1,47 @@
+describe('Test message service', () => {
+  let messageService
+  let rheaPromiseMock
+
+  beforeEach(async () => {
+    messageService = require('../server/services/message-service')
+    rheaPromiseMock = require('rhea-promise')
+    jest.mock('rhea-promise')
+  })
+
+  test('Message service sends the claim to two queues', async () => {
+    const claimRecord = {
+      claimId: 'MINE123',
+      propertyType: 'business',
+      accessible: false,
+      dateOfSubsidence: new Date(),
+      mineType: ['gold', 'iron']
+    }
+    const jsonData = JSON.stringify(claimRecord)
+    rheaPromiseMock._SendFunction.mockReset()

```

Comment:
`mockReset()` can be called in `beforeEach()`, rather than repeated in each test.

Reply:
It could, but in this case it doesn't make sense to do so, as the test redefines what that mock implementation does as part of that test. So it makes it clearer to have all the code in one place.